docs/images/contributor-flowchart.svg

```diff
@@ -0,0 +1,182 @@
+<text fill="#121619" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="637" y="703.5">work is not in scope</tspan></text>
+<text fill="white" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="578" y="645.5">Discuss your idea </tspan><tspan x="578" y="661.5">with maintainers</tspan></text>
+<text fill="white" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="528" y="764.5">Work with </tspan><tspan x="528" y="780.5">maintainers to </tspan><tspan x="528" y="796.5">integrate your </tspan><tspan x="528" y="812.5">feature into </tspan><tspan x="528" y="828.5">Qiskit</tspan></text>
+<text fill="white" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="636" y="763.5">Publish the </tspan><tspan x="636" y="779.5">feature yourself </tspan><tspan x="636" y="795.5">and consider </tspan><tspan x="636" y="811.5">joining the </tspan></text>
+<text fill="white" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px" text-decoration="underline"><tspan x="698.82" y="811.5"><a fill="white" href="https://qisk.it/qiskit-ecosystems">Qiskit </a></tspan><tspan x="636" y="827.5"><a fill="white" href="https://qisk.it/qiskit-ecosystems">Ecosystem</a></tspan></text>
+<path d="M683.5 736V713.09C683.5 710.328 681.261 708.09 678.5 708.09H633.5C630.739 708.09 628.5 705.851 628.5 703.09V681" stroke="#343A3F"/>
+<path d="M573.5 736V713.09C573.5 710.328 575.739 708.09 578.5 708.09H623.5C626.261 708.09 628.5 705.851 628.5 703.09V681" stroke="#343A3F"/>
+<path d="M572 735.999L573.5 739.999L575 735.999H572Z" fill="#343A3F"/>
+<line x1="573.5" y1="735.999" x2="573.5" y2="712.999" stroke="#343A3F"/>
+<path d="M682 735.999L683.5 739.999L685 735.999H682Z" fill="#343A3F"/>
+<line x1="683.5" y1="735.999" x2="683.5" y2="712.999" stroke="#343A3F"/>
+<rect x="511.5" y="475.5" width="110" height="81" fill="#6929C4" stroke="#491D8B"/>
+<rect x="630.5" y="475.5" width="119" height="81" fill="#6929C4" stroke="#491D8B"/>
+<text fill="white" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="521" y="493.5">Open a GitHub </tspan><tspan x="521" y="509.5">issue with </tspan><tspan x="521" y="525.5">feature request </tspan><tspan x="521" y="541.5">template</tspan></text>
+<text fill="white" xml:space="preserve" style="white-space: pre" font-family="IBM Plex Sans" font-size="12" letter-spacing="0.32px"><tspan x="637" y="494.5">Open a GitHub </tspan><tspan x="637" y="510.5">discussion in the </tspan></text>
```

Comment:
I would go for something shorter:
 * `Open a GitHub issue with feature request template` -> `Open a feature request issue`
 * `Open a GitHub discussion in the [qiskit-community/feedback](link) repo` -> * `Open [a discussion](link)`

Reply:
I prefer using the full name of the repo for people searching in github

Reply:
✅ done for feature request